db/migrate/20230728192141_add_lifted_to_project.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddLiftedToProject < ActiveRecord::Migration[5.2]
+  def change
+    add_column :projects, :lifted, :boolean, default: false
```

Comment:
Strong migrations recommends this be a two step operation: https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value

Although I'm not sure if you can have a boolean column without it defaulting to something...

```
class AddLiftedToProject < ActiveRecord::Migration[5.2]
  def up
    add_column :projects, :lifted, :boolean
    change_column_default :projects, :lifted, false
  end

  def down
    remove_column :projects, :lifted
  end
end
```

Reply:
thanks, just pushed up a fix